yoc/yoc-htb.js

```diff
@@ -342,6 +359,15 @@ function YocHtb(configs) {
             var pixelUrl = Browser.getProtocol() + '//t.visx.net/push_sync?wrapperType=IX&wrapperVersion='
                 + SpaceCamp.version + '&adapterVersion=' + __profile.version;
 
+            var gdprStatus = ComplianceService.gdpr.getConsent();
```

Comment:
Missing system tests to validate request contains `gdpr` data. Please refer to the following doc for more information: https://kb.indexexchange.com/ix_library_partners/build_adapter/test_case_reference.htm#validateBidRequestWithTcf2

Reply:
Test had been added